index.html

```diff
@@ -20,24 +20,128 @@
+          <li><img src="./Exported_elements/Images/Counter.png" alt="bullet" class="bullet">2015</li>
+        </ul>
+      </header>
+      <p>A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+      <ul class="tags">
+        <li><p>html</p></li>
+        <li><p>css</p></li>
+        <li><p>javaScript</p></li>
+      </ul>
+      <button type="button">See Project</button>
+    </div><br>
+    <div class="card">
+      <img src="./Exported_elements/Images/Snapshoot_Portfolio_3.png" alt="portfolio img" class="portfolio-img">
+      <header class="work-header">
+        <h1>Tonic</h1>
+        <ul>
+          <li class="canopy">CANOPY</li>
+          <li><img src="./Exported_elements/Images/Counter.png" alt="bullet" class="bullet">Back End Dev</li>
+          <li><img src="./Exported_elements/Images/Counter.png" alt="bullet" class="bullet">2015</li>
+        </ul>
+      </header>
+      <p>A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+      <ul class="tags">
+        <li><p>html</p></li>
+        <li><p>css</p></li>
+        <li><p>javaScript</p></li>
       </ul>
+      <button type="button">See Project</button>
+    </div>
+    <div class="card">
+      <img src="./Exported_elements/Images/Snapshoot_Portfolio_4.png" alt="portfolio img" class="portfolio-img">
+      <header class="work-header">
+        <h1>Multi-Post<br>Stories</h1>
+        <ul>
+          <li class="canopy">CANOPY</li>
+          <li><img src="./Exported_elements/Images/Counter.png" alt="bullet" class="bullet">Back End Dev</li>
+          <li><img src="./Exported_elements/Images/Counter.png" alt="bullet" class="bullet">2015</li>
+        </ul>
+      </header>
+      <p>A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+      <ul class="tags">
+        <li><p>html</p></li>
+        <li><p>css</p></li>
+        <li><p>javaScript</p></li>
+      </ul>
+      <button type="button">See Project</button>
+    </div>
+  </section>
+  <section class="about-info">
+    <h1>About<br>Myself</h1>
+    <p>Hello I’m a software developer! I can help you build a product , feature or website<br> Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <h3>LET'S CONNECT</h3> 
+    <ul class="socials">
+      <li>
+        <a href="#"><i class="fab fa-twitter fa-2x"></i></a>
+      </li>
+      <li>
+        <a href="#"><i class="fab fa-linkedin-in fa-2x"></i></a>
+      </li>
+      <li>
+        <a href="#"><i class="fab fa-medium-m fa-2x"></i></a>
+      </li>
+      <li>
+        <a href="#"><i class="fab fa-github fa-2x"></i></a>
+      </li>
+      <li>
+        <a href="#"><i class="fab fa-angellist fa-2x"></i></a>
+      </li>
+    </ul><br>
+    <button type="button" class="resume">Get my resume</button><br>
```

Comment:
- [ ] Please make sure to follow the Figma template as close as possible, your button is different than the one in the Figma template. :computer: 
## Figma
![image](https://user-images.githubusercontent.com/28109626/148598633-507747e5-3ac9-4d44-a0d9-b74bd12d4a13.png)
## Your Project
![image](https://user-images.githubusercontent.com/28109626/148598697-02cb7d20-44a9-4021-8345-51f083e9223c.png)